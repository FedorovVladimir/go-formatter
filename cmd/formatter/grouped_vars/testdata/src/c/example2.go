package main

var ( // want "grouped vars"
	c = "c"
)
var d = "d"
