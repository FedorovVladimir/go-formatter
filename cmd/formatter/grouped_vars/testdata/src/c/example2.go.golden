package main

var (
	c = "c"
	d = "d"
)
